pro_1.py

```diff
@@ -0,0 +1,92 @@
+import json
+from re import A
+import re
+
+database_file=open("database.txt","r+")
+dict_n=json.load(database_file)
+def loged_in():
+    print("succes")
+
+def login():
+    print("login page")
+    return
+def signup():
```

Comment:
Split each step into a different method, let's say we have SignUpView
this signUpView containts
def start..
def read_name
def read_email
def read_birthdate
def ....